lightcone/plot_3d.py

```diff
@@ -317,3 +328,44 @@ def match_2d_plot_to_3d_plot(self, fig, plot_2d, alpha=1):
         ax.invert_xaxis()
 
         return fig
+
+    def color_lens_and_source(self, config_handler, sample, noised_image=True, artifact_image=False):
+        """
+        Colors the source galaxy blue and the lens galaxy yellow
+
+        :param config_handler: ConfigHandler object used for Paltas configuration file. Need to import ConfigHandler
+         from paltas.Configs.config_handler and initialize the ConfigHandler
+        :param sample: dict of config_handler.get_current_sample()
+        :param noised_image: if True, return the colored noised image via self.sim_source_with_noise()
+        :param artifact_image: if True, return the artifacted image. Same image as Zl from self.plot3d()
+        :return: numpy.ndarray of either the noised image, artifacted image, or both, with the source colored blue and
```

Comment:
what does artifact image mean here? Not so sure I understand from the documentation